setuptools_rust/__init__.py

```diff
@@ -7,6 +7,8 @@
 from .test import test_rust
 from .tomlgen import tomlgen_rust, find_rust_extensions
 from .utils import Binding, Strip
+from .version import __version__
```

Comment:
From testing I think this needs to be:

```suggestion
from .version import version as __version__
```

... does that match your experience too?

Reply:
Oops, I initially used a template with ``__version__``. I removed the template and forgot to re-generate ``version.py`` before I pushed the change.

Reply:
👍 yeah I considered adding a template but thought the default was nicer too.